runtime/compiler/trj9/control/CompilationThread.cpp

```diff
@@ -10319,26 +10330,37 @@ TR::CompilationInfo::computeFreePhysicalMemory(bool &incompleteInfo)
          if (0 == j9sysinfo_get_memory_info(&memInfo)
              && memInfo.availPhysical != OMRPORT_MEMINFO_NOT_AVAILABLE)
             {
-            uint64_t cached = (memInfo.cached == OMRPORT_MEMINFO_NOT_AVAILABLE ? 0 : memInfo.cached);
-#if defined(LINUX)
-            uint64_t buffered = (memInfo.buffered == OMRPORT_MEMINFO_NOT_AVAILABLE ? 0 : memInfo.buffered);
-#else
-            uint64_t buffered = 0;
-#endif
-
-            if (!incomplete)
+            uint64_t freePhysicalMemorySizeB = memInfo.availPhysical;
+           
+            TR_VerboseLog::writeLineLocked(
+               TR_Vlog_COMPSTART,
+               "Harry memInfo.availPhysical=%llu MB",
+               freePhysicalMemorySizeB >> 20
+               );
+ 
+            if (memInfo.cached != OMRPORT_MEMINFO_NOT_AVAILABLE)
+               {
+               TR_VerboseLog::writeLineLocked(
+                  TR_Vlog_COMPSTART,
+                  "Harry memInfo.cached=%llu KB",
+                  memInfo.cached >> 10
+                  );
+               freePhysicalMemorySizeB += memInfo.cached;
+               }
+            else
                {
-               if (memInfo.cached == OMRPORT_MEMINFO_NOT_AVAILABLE)
-                  incomplete = true;
+               freePhysicalMemorySizeB = OMRPORT_MEMINFO_NOT_AVAILABLE;
```

Comment:
I would not use `freePhysicalMemorySizeB = OMRPORT_MEMINFO_NOT_AVAILABLE;` here just in case is interested in partial information. The statement above would prevent any other queries to the physical memory query.

Reply:
Yes the debug statements are only temporary. After my initial change, I'm still getting the same old numbers so I want to print these out for debugging purposes. Okay, I will remove the `freePhysicalMemorySizeB = OMRPORT_MEMINFO_NOT_AVAILABLE`s.